yoti-sdk-parent/pom.xml

```diff
@@ -50,17 +50,38 @@
     <url>http://github.com/getyoti/yoti-java-sdk.git</url>
   </scm>
   
-  <distributionManagement>
-    <snapshotRepository>
-      <id>ossrh</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-    </snapshotRepository>
-    <repository>
-      <id>ossrh</id>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-    </repository>
-  </distributionManagement>
```

Comment:
I'd leave this here or activate the sonatype profile by default (alternatively the nexus could be the default, up to what you think best).

It's always better to have a default (usually you'd pick the one that get's used more often) and then have profiles for all other cases. Otherwise it forces users to always have to include a profile arg when running the mvn command.

Reply:
I'd rather not.  We only need the profiles if we specifically want to deploy to either sonatype or nexus, and doing either should not be frequent.  In this case I'm quite happy to be slightly awkward about it - they'd have to be following a release process in the wiki anyway.